BDArmory/Radar/VesselRadarData.cs

```diff
@@ -1133,10 +1188,12 @@ private void WindowRadar(int windowID)
                     bool islocked = availableRadars[i].locked;
                     float currentAngle = availableRadars[i].currentAngle;
 
-                    float radarAngle = VectorUtils.SignedAngle(projectedVesselFwd,
-                    availableRadars[i].referenceTransform.forward.ProjectOnPlanePreNormalized(referenceTransform.up),
-                        referenceTransform.right);
-
+                    availableRadars[i].UpdateDisplayTransform();
```

Comment:
These computations should be in `LateUpdate` to avoid being done multiple times per frame.

Reply:
This stuff is gonna take a while to sort out... ah... legacy code...